docs/specifications/tests/DNSSEC-TP/dnssec13.md

```diff
@@ -1,150 +1,177 @@
-   2. Send the SOA query over UDP.
-      1. If no DNS response is returned, then output
-         *[NO_RESPONSE]* and go to next name server IP address.
-      2. Else, if the DNS response contains no SOA record in the
-         answer section, then output *[NO_RESPONSE_RRSET]* and go to
-         the next name server IP address.
+               the algorithm then add name server IP and DNSKEY algorithm to the
+               *Algo not signed DNSKEY* set.
+
+   2. Send *SOA Query* over UDP.
+      1. If no DNS response is returned, then go to next name server IP.
+      2. Else, if the RCODE of the DNS response is not "NOERROR" go to next name
+         server IP.
+      3. Else, if the AA flag of the response is unset, then go to the next name
+         server IP.
+      4. Else, if the DNS response contains no SOA record in the answer
+         section, then go to the next name server IP.
+      3. Else, if the DNS response contains no RRSIG for the SOA RRset, then
+         go top the next name server IP.
       3. Else, do:
-         1. Extract all RRSIG records from the response.
-         2. If there is no RRSIG for SOA, then output
-            *[RRSET_NOT_SIGNED]* and go to next name server IP
-            address.
-         3. For each algorithm in *DNSKEY Algorithm* do:
+         1. Extract all RRSIG records for the SOA RRset from the response.
+         5. For each algorithm in *DNSKEY Algorithm* do:
             1. If there is no RRSIG for the SOA RRset created by
-               the algorithm then output *[ALGO_NOT_SIGNED_RRSET]*.
-         4. For each RRSIG do:
-            1. If the key ID from the RRSIG is not a member of the
-               set *DNSKEY Key ID*, then output
-               *[RRSIG_NOT_MATCH_DNSKEY]* and go to next RRSIG.
-            2. If the RRSIG cannot be verified by the DNSKEY from
-               *DNSKEY Record* with the matching RRSIG key ID, then
-               output *[RRSIG_BROKEN]*.
-   3. Send the NS query over UDP.
-      1. If no DNS response is returned, then output
-         *[NO_RESPONSE]* and go to next name server IP address.
-      2. Else, if the DNS response contains no NS record in the
-         answer section, then output *[NO_RESPONSE_RRSET]* and go to
-         next name server IP address.
+               the algorithm then add name server IP and DNSKEY algorithm to the
+               *Algo not signed SOA* set.
+
+   3. Send *NS Query* over UDP.
+      1. If no DNS response is returned, then go to next name server IP.
+      2. Else, if the RCODE of the DNS response is not "NOERROR" go to next name
+         server IP.
+      3. Else, if the AA flag of the response is unset, then go to the next name
+         server IP.
+      4. Else, if the DNS response contains no SOA record in the answer
+         section, then go to the next name server IP.
+      3. Else, if the DNS response contains no RRSIG for the NS RRset, then
+         go top the next name server IP.
```

Comment:
Could you regroup in a sub listing? (and maybe align with other PRs with a link to the IANA Rcode list)